packages/example/components/chains/babylon/example.tsx

```
@@ -0,0 +1,703 @@
+import { Verifier } from 'bip322-js';
+import { Input } from '../../ui/input';
+import { createPSBT } from '../btc/utils';
+
+function removeNull(obj: any): any {
+  if (obj !== null && typeof obj === 'object') {
+    // eslint-disable-next-line @typescript-eslint/no-unsafe-argument
+    return Object.entries(obj)
+      .filter(([, v]) => v != null)
+      .reduce(
+        (acc, [k, v]) => ({
+          ...acc,
```

Biome (1.9.4)

[error] Avoid the use of spread (`...`) syntax on accumulators.

Spread syntax should be avoided on accumulators (like those in `.reduce`) because it causes a time complexity of `O(n^2)`.
Consider methods such as .splice or .push instead.

(lint/performance/noAccumulatingSpread)